src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsresettlementpassportapi/service/ServiceUtils.kt

```diff
@@ -142,3 +135,12 @@ fun convertFromNameToDeliusAuthor(prisonCode: String, name: String): DeliusAutho
     surname = if (splitName.size != 1) splitName.last() else "",
   )
 }
+
+fun convertToDeliusCaseNoteType(assessmentType: ResettlementAssessmentType) = when (assessmentType) {
+  ResettlementAssessmentType.BCST2 -> DeliusCaseNoteType.IMMEDIATE_NEEDS_REPORT
+  ResettlementAssessmentType.RESETTLEMENT_PLAN -> DeliusCaseNoteType.PRE_RELEASE_REPORT
+}
+
+tailrec fun getFibonacciNumber(n: Int, a: Int = 0, b: Int = 1): Long {
```

Comment:
Fancy, don't think I've ever seen anyone use `tailrec` in kotlin. Works nice for this though!